esmvaltool/recipes/recipe_ocean_temperature_cmip6.yml

```diff
@@ -0,0 +1,80 @@
+# ESMValTool
+# recipe_ocean_temperature_cmip6.yml
+---
+documentation:
+  description: |
+    Recipe to demonstrate a basic plots based on the monthly ocean
+    temperature in CMIP6.
+
+    Note that the time series will not function without the FX files, which
+    are not available yet on JASMIN (2019/04/15).
+
+    The maps will be plotted in their models native grids, which can sometimes
+    look strange. Regridding is a possible solution, but will not work without
+    the fx files, which are not yet available.
+
+  authors:
+    - demo_le
+
+  maintainer:
+    - demo_le
+
+  references:
+    - demora2018gmd
+
+  projects:
+    - ukesm
+
+
+datasets:
+     # CMIP6
+     - {dataset: CESM2,            project: CMIP6,    exp: historical, ensemble: r1i1p1f1, grid: gn, start_year: 2000,  end_year: 2014}
+     - {dataset: CNRM-CM6-1,       project: CMIP6,    exp: historical, ensemble: r1i1p1f2, grid: gn, start_year: 2000,  end_year: 2014}
+     - {dataset: GISS-E2-1-G,      project: CMIP6,    exp: historical, ensemble: r1i1p1f1, grid: gn, start_year: 2000,  end_year: 2014}
+     - {dataset: UKESM1-0-LL,      project: CMIP6,    exp: historical, ensemble: r1i1p1f2, grid: gn, start_year: 2000,  end_year: 2014}
+     - {dataset: MIROC6,           project: CMIP6,    exp: historical, ensemble: r1i1p1f1, grid: gn, start_year: 2000,  end_year: 2014}
+
+preprocessors:
+  # --------------------------------------------------
+  # Time series preprocessors
+  # --------------------------------------------------
+  # prep_timeseries_1: # For 2D fields
+  #   custom_order: true
+  #   average_region:
+  #     coord1: longitude
+  #     coord2: latitude
+  #   multi_model_statistics:
+  #     span: overlap
+  #     statistics: [mean ]
+
+  # --------------------------------------------------
+  # Maps preprocessors
+  # --------------------------------------------------
+  prep_map_1: # For Global 2D fields
+    time_average:
+
+diagnostics:
+  # --------------------------------------------------
+  # Time series diagnostics
+  # --------------------------------------------------
+  # diag_timeseries_1:
+  #   description: Global Ocean Surface mean timeseries
+  #   variables:
+  #     tos: # Temperature ocean surface
+  #       mip: Omon
+  #       preprocessor: prep_timeseries_1
+  #   scripts:
+  #     Global_Ocean_Surface_mean_timeseries:
+  #       script: ocean/diagnostic_timeseries.py
+  # --------------------------------------------------
+  # Map diagnostics
+  # --------------------------------------------------
+  diag_map_1:
+    description: Global Ocean Surface map
+    variables:
+      tos: # Temperature ocean surface
+        preprocessor: prep_map_1
+        mip: Omon
+    scripts:
+      Global_Ocean_Surface_mean_map:
+        script: ocean/diagnostic_maps.py
```

Comment:
cool! can you maybe comment out the diagnostic per se and just leave the preprocessor so that whatever it's in the recipe can be run by any given user who's got access to data? Also move it to recipes/examples because what I suggested would be just an example to show how to handle CMIP6 data, nothing science-oriented; no biggie if you leave it how it is anyway, I am going to write a whole lot of examples after the Easter hols and this will be one of them :beer: